src/mesh/mesh.hpp

```diff
@@ -149,9 +149,12 @@ class Mesh {
   int GetMaxLevel() const noexcept { return max_level; }
   int GetCurrentLevel() const noexcept { return current_level; }
   std::vector<int> GetNbList() const noexcept { return nblist; }
+  void SetAllowedDt(const Real dt) { max_allowed_dt_ = dt; }
+  Real NewDt() { return max_allowed_dt_; }
```

Comment:
Make method const.

Reply:
I went ahead and changed these to be consistent with the functions above them.  In general, though, I'm not a huge fan of needlessly marking functions `noexcept`.  `const` is a little better since it's a bit more than semantics.

Reply:
ended up removing these for other reasons.